packages/sfui/frameworks/react/components/SfButton/SfButton.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
button doesn't have to be interactive (it can be rendered as link)

Reply:
according to react docs, `forwardRef` has to be run on the client.

https://react.dev/reference/rsc/use-client#using-third-party-libraries